### DISCUSSION TITLE
Cats?

### DISCUSSION THREAD

**Discussion by FormSans** [Luke100000/minecraft-comes-alive#475] [Ideas]
As much as I am enjoying this mod so far, I find that there doesn't seem to be a way for cats to spawn naturally so far. If you make a big enough village, will cats start to spawn? If not, maybe add that so that cats can still spawn naturally?

**Comment by Luke100000** [comment]
Hmm, shouldn't this be covered by vanilla behavior? Anyways, since a lot of people asked for pets I think I know what's on my todo now.
